apps/wing-console/console/app/assets/openChrome.applescript

```diff
@@ -0,0 +1,95 @@
+(*
+Copyright (c) 2015-present, Facebook, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file at
+https://github.com/facebookincubator/create-react-app/blob/master/LICENSE
+*)
+
+property targetTab: null
+property targetTabIndex: -1
+property targetWindow: null
+property theProgram: "Google Chrome"
```

Comment:
What happens if I'm not using Chrome as my default browser or don't have it installed?

Reply:
It goes back to the default behavior. In my case, I use Arc and does not actually reuse the browser tab (sadly).